README.md

```diff
@@ -12,19 +12,98 @@ other components of the system, including the Istio mixer and the Istio proxy me
 If you have a question about the Istio Manager or have a problem using it, please
 [file an issue](https://github.com/istio/manager/issues/new).
 
-## Build instructions ##
+## Setting up a local dev environment ##
+
+_Note:_ This section applies to Mac and Windows users only.
+
+### Pre-requisites ##
+
+- Setup Go 1.7.5+ on your host machine
+- Clone this repository
+- Install [Virtualbox](https://github.com/kubernetes/minikube/releases)
+- Install [Minikube](https://github.com/kubernetes/minikube/releases)
+- Install [Vagrant](https://www.vagrantup.com/downloads.html)
+- Install [kubectl](https://kubernetes.io/docs/user-guide/prereqs/)
+
+### 1. Start Minikube
+
+    minikube config set kubernetes-version v1.5.2
```

Comment:
We shouldn't be hard-coding versions in many places. Versions move very fast in Kubernetes, and there is already 1.6 alpha available.